server/lib/nl/config_builder/base.py

```diff
@@ -75,6 +75,53 @@ def _replace_special(input_string_title_case: str) -> str:
   return input
 
 
+def _count_caps(word: str) -> int:
+  return sum(1 for c in word if c.isupper())
+
+
+def _make_sentence_case(input_string: str) -> str:
```

Comment:
Maybe this function warrants some unit tests?

Reply:
yup will do